nilearn/_utils/niimg.py

```diff
@@ -14,6 +14,17 @@
 from .compat import _basestring
 
 
+def get_data(img):
```

Comment:
This is a public function, use in our examples. Hence it needs a docstring.

As we are exposing such a public function, we should add to it the nilearn goodies: working with filenames and globbing. The easiest way to do this is probably to insert a check_niimg in the beginning of the function